doc/nrf/links.txt

```diff
@@ -213,6 +213,7 @@
 .. _`Memfault WebBluetooth Client`: https://memfault.github.io/web-ble-example/
 
 .. _`PSA Cryptography API 1.0.1`: https://armmbed.github.io/mbed-crypto/1.0.1/html/index.html
+.. _`PSA Certified Crypto API 1.1`: https://arm-software.github.io/psa-api/crypto/1.1/
```

Comment:
Likely unrelated to this PR but we are actively supporting PSA Crypto API version 1.2.0 
https://arm-software.github.io/psa-api/crypto/1.2/

As well as the PAKE extensions available here:
https://arm-software.github.io/psa-api/crypto/1.2/ext-pake/

Reply:
@frkv: Does this mean that I should be referring to PSA Crypto API 1.2 instead of 1.1 in line 137 in cryptography.rst as well?